test/test_device_thermal_generation_constructors.jl

```diff
@@ -634,7 +634,7 @@ end
     moi_tests(UC, false, 32, 0, 8, 4, 14, true)
     # Cbc can have reliability issues with SoS. The objective function target in the this
     # test was calculated with CPLEX do not change if Cbc gets a bad result
-    psi_checksolve_test(UC, [MOI.OPTIMAL], 1286.0, 10.0)
+    psi_checksolve_test(UC, [MOI.OPTIMAL], 8500.0, 10.0)
```

Comment:
The thing you ran in CPLEX was wrong formulation, so that results doesn't make sense and I constructed these tests myself so the value hasn't been changes since. this is the same values as before in v0.14

Reply:
ok, I ran the test with CPLEX before the PR with the cost function changes. The previous value is the current value in master. We had disabled it because of the segmentation faults in Cbc.